src/client/java/net/neoforged/neoforge/client/loading/NeoForgeLoadingOverlay.java

```diff
@@ -53,8 +54,6 @@ public NeoForgeLoadingOverlay(final Minecraft mc, final ReloadInstance reloader,
         this.reload = reloader;
         this.onFinish = errorConsumer;
         this.displayWindow = displayWindow;
-        var logoGpuTexture = (GlTexture) mc.getTextureManager().getTexture(MOJANG_STUDIOS_LOGO_LOCATION).getTexture();
```

Comment:
Hmm why is this file changed? What does it have to do with the event busses?

Reply:
> This FML update also includes the ELS theming rework.

Change as part of that update.